packages/block-editor/src/layouts/flex.js

```diff
@@ -162,3 +176,31 @@ function FlexLayoutJustifyContentControl( {
 		</fieldset>
 	);
 }
+
+function FlexWrapControl( { layout, onChange } ) {
+	const { flexWrap = flexWrapMap.wrap } = layout;
+	const helpTexts = {
+		wrap: __( 'Items wrap onto multiple lines.' ),
```

Comment:
```suggestion
		wrap: __( 'Items may wrap onto multiple lines.' ),
```

Just quick clarification that items won't _always_ wrap.

Reply:
The `allow` in the new label is similar to your suggestion, no?